### DISCUSSION TITLE
My college does not provide an email address, have been trying to get in touch with support for over a month now.

### DISCUSSION THREAD

**Discussion by hobi-design** [community/community#113474] [GitHub Education]
### Hi everyone,

Hi, I have been trying to join GitHub Education as a student. I am currently enrolled as a first year student at a university that unfortunately does not provide, or at least has not yet provided students of my batch with a college email address.

I have been trying to get in touch with GitHub staff for over a month regarding this but have not received any support. 

My support ticket: #2608568

My application has been rejected three times, I have submitted my student ID as well as **enrollment form with the email address that I am using with GitHub.**

Completely lost at this point. Not sure what else I can do. Can someone please get me in touch with GitHub support?

Thank you!

**Comment by SebastienXia** [comment]
## Hello, @OutOfTokens 

> You can use your personal email, the university email is not obligatory in the process.

`[close the thread and mark as answered if this helped you resolve your issue]`

**Reply by hobi-design** [reply]
That's what I am doing. The email is even on the enrollment form, but it keeps rejecting my application. :/

> What happened?
> The item you uploaded is insufficient to demonstrate your current academic status.
> Please see suggestions.
> Suggested proof to expedite your approval:
> 
> Attendees of this school must verify and use their academic email address.
> 
> Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
> 
> Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

I can't use my academic email address (wasn't provided with one) and I did capture my documents with the laptop camera.

**Comment by SebastienXia** [comment]
Of course the email which is associated with your Github Account. 😅

**Comment by samus-aran** [comment]
Hi @OutOfTokens 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

I do want to highlight an important point when you raise a ticket to request your school details to be updated: 

**Accepted types of evidence include a link to a publicly-accessible page on the school website or an affidavit from your school on school letterhead, detailing their current academic email issuing policy, or listing their academic domain(s), as it is relevant in your case.** 👈 

_Please note that screenshots of emails purportedly sent by the school to non-academic webmail addresses are not considered to be reliable supporting documentation._

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by hobi-design** [reply]
Hi @samus-aran I already went through the links you've shared. One of the first things the links suggest is to:

> If you are attending one of the schools that request specific verified academic emails and believe the school's details are incorrect please raise a ticket with Education support using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location".

I **raised a ticket with Education support** **over a month ago.** Here is my Ticket ID: 2608568

I have also tried to apply with me enrollment form as well as my college ID card. Both of which were rejected. If the Support staff can tell me what exact documents I need to present (that are actually issued by my university), I absolutely will.

**Comment by hobi-design** [comment]
Bump? I have already done everything that the Docs suggested me to try.
